tests/PHPUnit/Fixtures/ManyVisitsWithGeoIP.php

```diff
@@ -58,21 +58,20 @@ class ManyVisitsWithGeoIP extends Fixture
     public function setUp()
     {
         $this->setUpWebsitesAndGoals();
-        self::downloadGeoIpDbs();
```

Comment:
Are the geoip files provided in some other way during tests now? Or would I have to download them myself locally to run these tests?

Reply:
Nevermind, see the test files below. For the binary .mmdb file, would using LFS be a good idea?

Reply:
Theoretically it would be good to add them to LFS. But we currently skip downloading any LFS files on travis for all builds except the UI builds. That saves a lot time. But as the mmdb files are required for Integration and System tests, we would need to download them there as well. Also currently the mmdb files are quite small, so guess there's no need to move them to LFS.

Reply:
:+1: